requirements.txt

```diff
@@ -33,3 +33,8 @@ django-sendfile
 django-polymorphic
 networkx
 rdflib>=4.1.0
+celery-with-redis
+django-celery
+scikit-learn
+nilearn
+-e git://github.com/vsoch/pybraincompare.git@fe0efe78e23ffb44f709230198a442cb117c7f0a
```

Comment:
do we need that specific commit or are you plannig to provide a backwards compatible API?